src/views/tables/OrdersTable.tsx

```diff
@@ -210,6 +210,7 @@ const getOrdersTableColumnDef = ({
         label: stringGetter({ key: STRING_KEYS.ACTION }),
         isActionable: true,
         allowsSorting: false,
+        getCellValue: undefined,
```

Comment:
I think if the above typing works, we'd maybe be able to just not pass in an argument here